components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1467,6 +1470,36 @@ where
         }
     }
 
+    // If lease expired, we will send a noop read index to renew lease.
+    fn try_renew_leader_lease(&mut self) {
```

Comment:
1. Because this tick is not registered once after lease was renewed (which is hard to implement for now because lease will be renewed after every write). Then if we set it to `x` then the max period that a peer has no lease could be `x - self.raft_store_max_leader_lease / 4`
2. Read traffic is counted by other threads, so 

And according to our tests, the cost of this tick is acceptable (1%~2% raftstore CPU after disabling hibernate regions), so I think that although the implementation is not optimal,